tests/conftest.py

```diff
@@ -1805,6 +1805,18 @@ def cluster_teardown_finalizer():
                     kms.cleanup()
                 except Exception as ex:
                     log.error(f"Failed to cleanup KMS. Exception is: {ex}")
+            if ocsci_config.MULTICLUSTER.get("acm_cluster"):
+                try:
+                    from ocs_ci.utility.aws import AWS
+
+                    thanos_bucket_name = (
+                        f"dr-thanos-bucket-{config.ENV_DATA['cluster_name']}"
```

Comment:
```
 tests/conftest.py:1813:45: F821 undefined name 'config'
 ```
 Looks like here we will need to do manual cherry-pick and us ocsci_config as here in 4.16 we import only:
 ```
 from ocs_ci.framework import config as ocsci_config
```